puma.gemspec

```diff
@@ -14,6 +14,7 @@ Gem::Specification.new do |s|
   s.email = ["evan@phx.io"]
   s.executables = ["puma", "pumactl"]
   s.extensions = ["ext/puma_http11/extconf.rb"]
+  s.add_runtime_dependency "concurrent-ruby", "~> 1.0", ">= 1.3.1"
```

Comment:
Pretty sure we don't want to force this dependency on all Puma users :)

Reply:
Thx for the comment :)

I can totally remove it from the runtimedeps, you're right 👍

Users without `concurrent-ruby` available when setting `auto` would get a `LoadError`. Maybe that's already fine with proper documentation?

I thought about reraising the LoadError with a more descriptive error message: "Please add `concurrent-ruby` to your bundle to use WEB_CONCURRENCY=auto".

Wdyt?

Only checking if Concurrent is defined will lead to an `invalid value for Integer() Argumenterror`, which I guess is unexpected as well.

Reply:
Yes, the load error should be caught and a nice message should be displayed.

I'm not sure if this is the solution Rails project had in mind. WDYT @rafaelfranca?

Reply:
I added a short simple warning message if requiring fails.